examples/util/openvino/OpenVINOModelImporter.js

```diff
@@ -681,6 +737,90 @@ class OpenVINOModelImporter {
 
           opCode = this._nn.SOFTMAX;
         } break;
+        case 'Normalize': {
+          const input = node.inputs[0];
+          const inDims = input.shape();
+          const inputId = this._getTensorId(input);
+          console.log(`  input shape: [${input.shape()}]`);
+
+          const acrossSpatial = node.getInt('across_spatial');
+          const channelShared = node.getInt('channel_shared');
+          const eps = node.getFloat('eps');
+
+          if (acrossSpatial !== 0 || channelShared !== 0) {
+            throw new Error(`Normalize not support across_spatial ${across_spatial} channel_shared ${channel_shared}.`);
+          }
+
+          inputs.push(inputId);
+
+          const output = node.outputs[0];
+
+          if (i === graph.nodes.length - 1) {
```

Comment:
could you please elaborate why can skip the last `Normalize` layer?

Reply:
Only in this case, when the Normalize operation appears as the last operation， we can skip the Normalize operation. and I realized this operation as function in [examples/face_recognition/utils.js](https://github.com/intel/webml-polyfill/pull/984/files?file-filters%5B%5D=.html&file-filters%5B%5D=.jpg&file-filters%5B%5D=.js#diff-8095bd061b3d471362cc188b106f9bec) of PR #984.

Reply:
For this case, I would suggest to extract the `Normalization` layer from your model instead adding `Normalization` support.

Reply:
I delete Normalization supportting.